README.md

```diff
@@ -46,85 +46,11 @@ A community maintained open source project aimed at making personal portfolio fo
 
 ## Contribution Guidelines🏗
 
-Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and improve it further
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and improve it further.
 
-To start contributing, follow the below guidelines:
+Click the image below for Set up and Contribution Guidelines:
 
-**1.** Fork [this](https://github.com/smaranjitghose/awesome-portfolio-websites) repository.
-
-**2.** Clone your forked copy of the project.
-
-```
-git clone --depth 1 https://github.com/<your_user_name>/awesome-portfolio-websites.git
-```
-
-<img src="./readme_assets/CloneRepo.png" width="500">
-
-**3.** Navigate to the project directory :file_folder: .
-
-```
-cd awesome-portfolio-websites
-```
-
-**4.** Add a reference(remote) to the original repository.
-
-```
-git remote add upstream https://github.com/smaranjitghose/awesome-portfolio-websites.git
-```
-
-**5.** Check the remotes for this repository.
-
-```
-git remote -v
-```
-
-**6.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
-
-```
-git pull upstream master
-```
-
-**7.** Create a new branch.
-
-```
-git checkout -b <your_branch_name>
-```
-
-**8.** Perform your desired changes to the code base.
-
-<p align="center"><img width=35% src="https://media2.giphy.com/media/L1R1tvI9svkIWwpVYr/giphy.gif?cid=ecf05e47pzi2rpig0vc8pjusra8hiai1b91zgiywvbubu9vu&rid=giphy.gif"></p>
-
-**9.** Track your changes:heavy_check_mark: .
-
-```
-git add .
-```
-
-**10.** Commit your changes .
-
-```
-git commit -m "Relevant message"
-```
-
-**11.** Push the committed changes in your feature branch to your remote repo.
-
-```
-git push -u origin <your_branch_name>
-```
-
-**12.** To create a pull request, click on `compare and pull requests`. Please ensure you compare your feature branch to the desired branch of the repo you are suppose to make a PR to.
-
-<img src="./readme_assets/ComparePR.png" width=600>
-
-**13.** Add appropriate title and description to your pull request explaining your changes and efforts done.
-
-**14.** Click on `Create Pull Request`.
-
-<img src="./readme_assets/CreatePR.png" width=600>
-
-**15** Voila :exclamation: You have made a PR to the awesome-developer-portfolio project :boom: . Sit back patiently and relax while the project maintainers review your PR. Please understand at times the time can vary from a few hours to a few days
-
-<p align="center"><img src="https://media.giphy.com/media/5mCQOcUfywmyI/giphy.gif" width=35%></p>
+[img src="./readme_assets/Set up and Contribution Guidelines.png"](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/CONTRIBUTING.md)
```

Comment:
This does not work!

Look up how to insert image in markdown with a link please

Reply:
🤨